src/app_controller.js

```diff
@@ -723,7 +734,8 @@ AppController.prototype.init = function() {
     BlocklyStorage.retrieveXml(window.location.hash.substring(1),
                                BlockFactory.mainWorkspace);
   } else {
-    BlockFactory.showStarterBlock();
+    this.createBlocklyInitPopup(true);
+    //this.BlockManagerView.createBlocklyInitPopup(true);
```

Comment:
In progress? If so, add end of line comment explaining what this is for. Otherwise delete.